default.yml

```diff
@@ -5,17 +5,19 @@
 controls: # Controls added to "No team"
   macos_settings:
     custom_settings:
-    - path: ../lib/macos-os-updates.ddm.json
-    - path: ../lib/macos-password.mobileconfig
+      - path: ./lib/macos-password.mobileconfig
+#     DDM support coming soon
+#     - path: ../lib/macos-os-updates.ddm.json
   windows_settings:
     custom_settings:
-    - path: ../lib/windows-screenlock.xml
+    - path: ./lib/windows-screenlock.xml
   scripts:
-  - path: ../lib/collect-fleetd-logs.sh
+    - path: ./lib/collect-fleetd-logs.sh
 queries:
-- path: ./lib/collect-fleetd-update-channel.queries.yml
+- path: ./lib/collect-fleetd-update-channels.queries.yml
```

Comment:
Do we need to to add an indent here before each query? (`- path`) I see you added indents for `custom_settings`.